### PR TITLE
Improve VisKart button

### PR DESCRIPTION
**Pull request by Arashfa0301**
Improved the design and the placement of "Vis kart" button at event pages. 

--------------
image

image

### CODE REVIEW

**Comment by ivarnakken** [comment]
I'm not a fan of the button being so eye-catching. With the current flat button, it blends in more nicely. Could it be an idea to just make the text itself clickable, and give it some other styling to indicate this? 

Also, the logo and text would need more spacing in between.

File: app/routes/events/components/EventDetail/EventDetail.css [hunk 1]
```diff
@@ -48,6 +48,13 @@
   align-items: center;
 }
 
+.iconWithTextMazeMap {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  flex-wrap: wrap;
+}
```
Comment (ivarnakken) [inline app/routes/events/components/EventDetail/EventDetail.css:56]: This is just `<Flex wrap alignItems="center" gap={20}  />`

File: app/routes/events/components/EventDetail/EventDetail.css [hunk 2]
```diff
@@ -58,3 +65,10 @@
 .registeredBox {
   margin-top: 1em;
 }
+
+.mapButton {
+  margin-top: 5px;
+  margin-bottom: 5px;
+  max-width: 130px;
+  max-height: 33px;
```
Comment (ivarnakken) [inline app/routes/events/components/EventDetail/EventDetail.css:73]: Why is this needed?

File: app/routes/events/components/EventDetail/index.tsx [hunk 3]
```diff
@@ -395,29 +398,34 @@ export default class EventDetail extends Component<Props, State> {
...
+                  <Button
+                    className={styles.mapButton}
                     onClick={() =>
                       this.setState({
                         mapIsOpen: !this.state.mapIsOpen,
                       })
                     }
                   >
-                    {' '}
+                    <CircularPicture size={25} alt="" src={mazemapLogo} />
```
Comment (ivarnakken) [inline app/routes/events/components/EventDetail/index.tsx:415]: Just use the `<Image />` component. This logo is not circular.

Comment (ivarnakken) [reply app/routes/events/components/EventDetail/index.tsx:415]: Also, never do `alt=""`

File: app/routes/events/components/EventDetail/EventDetail.css (hunk 1 above)
Comment (Arashfa0301) [reply app/routes/events/components/EventDetail/EventDetail.css:56]: True

File: app/routes/events/components/EventDetail/EventDetail.css (hunk 2 above)
Comment (Arashfa0301) [reply app/routes/events/components/EventDetail/EventDetail.css:73]: Forgot to delete:)

**Comment by LudvigHz** [comment]
> I'm not really a fan of the button being so eye-catching

I disagree. The current text is really hidden and not discoverable. The button is eye-catching but that's what we want. People will click on it and know what it's for in other events. It doesn't really matter if their attention is drawn for a bit. The info is there for them to see if they want and choose. And the mazemap logo makes it obvious what kind of map it is.

It could be made a bit smaller though if that's desirable.

File: app/routes/events/components/EventDetail/index.tsx (hunk 3 above)
Comment (Arashfa0301) [reply app/routes/events/components/EventDetail/index.tsx:415]: done

**Review by LudvigHz — APPROVED** [review]
LGTM

I like it! Could make the logo smaller if people find it too much, but I think it's an improvement :)

**Comment by Arashfa0301** [comment]
> LGTM
> 
> I like it! Could make the logo smaller if people find it too much, but I think it's an improvement :)

**Comment by Arashfa0301** [comment]
Clicked the wrong button 🥲

**Comment by Arashfa0301** [comment]
> LGTM
> 
> I like it! Could make the logo smaller if people find it too much, but I think it's an improvement :)

The logo looks smaller when the page is not zoomed in. It anyhow can be improved at a later date.
